xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -237,6 +237,15 @@ namespace PVR
      */
     std::string EpisodeName(void) const { return m_strShowTitle; };
 
+    /*!
+    * @brief Set m_genre to plugin proved sting when iGenreType = EPG_GENRE_USE_STRING
```

Comment:
let me make suggestion for the complete section. I would write it like this:
<pre>
    * @brief set the genre for this recording.
    * @param iGenreType The genre type ID. If set to EPG_GENRE_USE_STRING, set genre to the value provided with strGenre. Otherwise, compile the genre string from the values given by iGenreType and iGenreSubType
    * @param iGenreSubType The genre subtype ID
    * @param strGenre The genre
</pre>